ffi/src/lib.rs

```diff
@@ -14,9 +14,10 @@ use delta_kernel::expressions::{BinaryOperator, Expression, Scalar};
 use delta_kernel::schema::{ArrayType, DataType, MapType, PrimitiveType, StructType};
 use delta_kernel::snapshot::Snapshot;
 use delta_kernel::{DeltaResult, Engine, Error};
+use delta_kernel_ffi_macros::handle_descriptor;
 
-mod handle;
-use handle::{ArcHandle, SizedArcHandle, Unconstructable};
+pub mod handle;
```

Comment:
Do we expect things outside of the ffi crate to implement `Handles`? If not, I think we can `pub(crate)` this

Reply:
No. Good point!

Reply:
Hmm, making it `pub(crate)` suppresses docs for it tho... seems like it would be nice to have docs even for internal classes, for benefit of kernel developers?

Reply:
well, if you add `--document-private-items` to `cargo doc` it will generate those docs.

not _ideal_ for people who want to hack on kernel, but we could probably document that.